internal/driver/helper.go

```diff
@@ -14,23 +14,17 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/v3/errors"
 )
 
-const (
-	redactedStr = "//<redacted>@"
-)
-
-var (
-	userPassRegex = regexp.MustCompile(`//(\S+):(\S+)@`)
-)
+var userPassRegex = regexp.MustCompile(`//(\S+):(\S+)@`)
 
 type EdgeXErrorWrapper struct{}
 
 func (e EdgeXErrorWrapper) CommandError(command string, err error) errors.EdgeX {
 	return errors.NewCommonEdgeX(errors.KindServerError, fmt.Sprintf("failed to execute %s command", command), err)
 }
 
-// redact removes all instances of basic auth (ie. rtsp://username:password@server) from a url
+// redact removes all instances of basic auth (i.e. rtsp://username:password@server) from an url
```

Comment:
`a url` is the correct grammer here: https://www.techtarget.com/whatis/feature/Which-is-correct-a-URL-or-an-URL

Reply:
Even the silly IDE suggested to me that it should be `an` :-)